app.py

```diff
@@ -98,8 +98,8 @@ def role_loader(identity):
 
     # checking if the token's jti (jwt id) is in the set of revoked tokens
     # this check is applied globally (to all routes that require jwt)
-    @app.jwt.token_in_blacklist_loader
-    def check_if_token_in_blacklist(decrypted_token):
+    @app.jwt.token_in_blocklist_loader
+    def check_if_token_in_blacklist(jwt_header, decrypted_token):
```

Comment:
Is this correct? Should the a be an o?

Reply:
Sorry, that was the old function name and I left it the same when I updated the decorator.
Changed it now.